src/partials/header-content.hbs

```diff
@@ -9,4 +9,16 @@
             {{/if}}
         </div>
     </nav>
+    <div class="searchbar">
+      <form>
+        <div>
+          <input class="search-input" type="text" placeholder="">
+          <i class="fa fa-search"></i>
+        </div>
+        <div>
+          <input type="checkbox" id="scales" name="scales">
+          <label for="scales">Nur Developers durchsuchen</label>
+        </div>
```

Comment:
We may want to put a condition around this. I think displaying this checkbox only makes sense if there's more than one component, so we could maybe check the number of elements in `site.components` to decide whether to display this filter element or not?
https://docs.antora.org/antora-ui-default/templates/

<img width="815" alt="Screenshot 2022-02-09 at 14 05 20" src="https://user-images.githubusercontent.com/43753494/153207001-b87ff458-fa94-4ed6-be17-3bcf7996fc26.png">